cmd/proxy/main.go

```diff
@@ -79,6 +81,11 @@ type ProxyRunOptions struct {
 	agentPort uint
 	// Port we listen for admin connections on.
 	adminPort uint
+
+	// ID of this server.
+	serverID string
```

Comment:
What is the serverID set to when serverCount > 1?

Reply:
Server count is the number of servers in the HA setup.

ServerID is the id of this particular instance.

Do you have suggestions on how to make it more clear?

Reply:
proxyServerID maybe? Knowing its an HA cluster made we wonder if this might have to do with the Kube API Server.